packages/transaction-controller/src/types.ts

```diff
@@ -86,16 +86,6 @@ type TransactionMetaBase = {
    */
   currentTokenBalance?: string;
 
-  /**
-   * Unique ID for custodian transaction.
-   */
-  custodyId?: string;
```

Comment:
Minor, to simplify the release. Can we pre-emptively add this to the `CHANGELOG.md`?

Reply:
Done! I've added it under `Unreleased` - I'm not sure if that's right (I'm only used to fixing up changelogs at the last minute 🙂  )